synapse/rest/client/v2_alpha/keys.py

```diff
@@ -173,20 +183,24 @@ def __init__(self, hs):
         self.auth = hs.get_auth()
         self.device_handler = hs.get_device_handler()
 
+    @opentracing.trace
```

Comment:
does this need an operation name?

Reply:
It actually no longer needs to be wrapped now that servlets are traced by default.